gateway/sqlite.go

```diff
@@ -26,20 +26,85 @@ var (
 	errDBClosed         = errors.New("error gateway is closed")
 )
 
+// Node defines a lorawan node device.
+type gatewayNode struct {
```

Comment:
I don't understand the distinction between a deviceInfo and a gatewayNode

Is a `gatewayNode` stored in the DB? If not, why is it in the `gateway/sqlite.go` file?

Reply:
gatewayNode's information will all be stored in the db at some point. as we progress with the migration ideally this struct will be removed, and I placed it in sqlite.go so it would be easier to track what fields it should own.